README.adoc

```diff
@@ -307,17 +334,36 @@ output:
  T E S T S
 -------------------------------------------------------
 Running it.io.openliberty.guides.consumingrest.ConsumingRestTest
-Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.957 sec - in it.io.openliberty.guides.consumingrest.ConsumingRestTest
+Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.59 sec - in it.io.openliberty.guides.consumingrest.ConsumingRestTest
 
 Results :
 
-Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
+Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
+
 ----
 
+== Conclusions
+
+This guide solely focused on deserialization of JSON resources.  JSON-B provides the
+ability to *directly* serialize and deserialize POJOs.  This powerful API gives
+developers many options for working with JSON resources.
+
+In contrast, you may have noticed that the JSON-P implementation of this guide required
+the use of helper methods to process the JSON response.  This tactic is more
+straightforward, but can be cumbersome with more complex classes.
+
+After seeing these two ways to consume the returned JSON resources (JSON-B and JSON-P)
+from a RESTful web service you may be asking yourself, "Which way is better?"
+
+JSON-B was built on top of the existing JSON-P API.  It can do everything that JSON-P can
+and allows for more customization for serializing and deserializing.  We would recommend,
+if you have not already, to try using JSON-B for your next project.  If you want
+more information on JSON-B and how to serialize POJOs into JSON strings please refer to the
+http://json-b.net/users-guide.html[JSON Binding 1.0 Users Guide].
```

Comment:
Include only one space between sentences.

Reply:
Maybe under "What you'll learn" you could include a "Differences between JSON-B and JSON-P" section.

Suggested changes:

JSON-B allows you to directly serialize and deserialize POJOs. This API gives you options for working with JSON resources.

In contrast, JSON-P requires you to use helper methods to process the JSON response. This tactic is more straightforward, but it can be cumbersome with more complex classes.

JSON-B is built on top of the existing JSON-P API. JSON-B can do everything that JSON-P can and allows for more customization for serializing and deserializing. For more information on JSON-B and how to serialize POJOs into JSON strings, see the http://json-b.net/users-guide.html[JSON Binding 1.0 Users Guide].